### PR TITLE
recipe for tqdm-multiprocess

### PR DESCRIPTION
**Pull request by mediocretech**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/tqdm-multiprocess/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "tqdm-multiprocess" %}
+{% set version = "0.0.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # wheel...the setup.py file assumes a local requirements-dev.text
+  url: https://files.pythonhosted.org/packages/25/7e/0d889fc6c84e3df6b69aaafe893fc77f69b3d968ac9ce574d1c62c688050/tqdm_multiprocess-0.0.11-py3-none-any.whl
```
Comment (xhochy) [inline recipes/tqdm-multiprocess/meta.yaml:10]: Please update this to a pattern that looks similar to the one for sdists.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/tqdm-multiprocess/meta.yaml (hunk 1 above)
Comment (mediocretech) [reply recipes/tqdm-multiprocess/meta.yaml]: Thanks @xhochy

This should have been marked as `draft` before, but is ready now for review.

File: recipes/tqdm-multiprocess/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+{% set version = "0.0.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tqdm-multiprocess-{{ version }}.tar.gz
+    sha256: a74002a1222ea9cbe8cdc9bd460108c6009be359621fbee9b92d0515d4d180f7
+  - url: https://raw.githubusercontent.com/EleutherAI/tqdm-multiprocess/master/requirements-dev.txt
```
Comment (xhochy) [inline recipes/tqdm-multiprocess/meta.yaml:11]: This should also be conditional on the version.
```suggestion
  - url: https://raw.githubusercontent.com/EleutherAI/tqdm-multiprocess/{{ version }}/requirements-dev.txt
```

Comment (mediocretech) [reply recipes/tqdm-multiprocess/meta.yaml:11]: @xhochy I do not disagree, however this package has no official releases or tags in GitHub.  https://github.com/EleutherAI/tqdm-multiprocess/tags

Comment (xhochy) [reply recipes/tqdm-multiprocess/meta.yaml:11]: 😭

**Review by xhochy — APPROVED** [review]

**Pull request merged**